app/views/transactions/index.html.erb

```diff
@@ -0,0 +1,40 @@
+<div class="navbar bg-primary text-white">
+  <div class="nav-btn">
+    <%= link_to categories_path, class: 'btn btn-primary' do %>
+      <i class="fas fa-arrow-left"></i>
+    <% end %>
+  </div>
+  <div class="text-center px-4 fs-4">
+    <h2>Transactions</h2>
+  </div>
+</div>
+
+<div class="card-header">
+  <h1 class="card-title">Transactions for <%= @category.name %></h1>
+</div>
+
+<div class="total-transactions">
+  <h2>Total Amount: $<%= @total_amount %></h2>
+</div>
+
+<div class="transaction-list">
+  <% @category.transactions.order(created_at: :desc).each do |transaction| %>
+    <div class="transaction-row">
+      <div class="transaction-column">
+        <div class="hr border-primary">
+          <%= transaction.name %>
+          <p class="text-muted">Created on <%= transaction.created_at.strftime('%B %d, %Y') %></p>
+        </div>
+      </div>
+      <div class="transaction-column">
+        <div class="hr">
+          $<%= transaction.amount %>
+        </div>
+      </div>
+    </div>
+  <% end %>
+</div>
```

Comment:
- [ ] You have done a wonderful job of adding dates to the transactions and ensuring they are ordered by the most recent, however, there are still some design changes mentioned by the previous reviewer, left to be made. Kindly ensure your transaction page matches the given design as seen below:

### Your design
![Screenshot 2023-12-08 at 23 09 36 (2)](https://github.com/mumbereh/Budget_App/assets/64774936/16d01cb5-7d28-49f6-ad7f-312027fe04cf)

### The given design
<img width="354" alt="Screenshot 2023-12-08 at 23 09 36" src="https://github.com/mumbereh/Budget_App/assets/64774936/3c997603-6d47-408b-98f1-541dd41f6ded">

- [ ] Kindly make the title of the page `Transactions` instead of giving it the title of one transaction. e.g. `Transactions for food` as it is meant to show all transactions, not only one.